docs/mongocli/command/mongocli-cloud-manager-logs-decrypt.txt

```diff
@@ -0,0 +1,88 @@
+.. _mongocli-cloud-manager-logs-decrypt:
+
+===================================
+mongocli cloud-manager logs decrypt
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Decrypts a log file with the provided local key file or KMIP files.
+
+Syntax
+------
+
+.. code-block::
+
+   mongocli cloud-manager logs decrypt [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -f, --file
+     - string
+     - true
+     - Path to the file that contains encrypted audit logs.
```

Comment:
Is it an absolute path?

Reply:
either absolute or relative to the location where the cli runs